docs/configuration.md

```diff
@@ -176,9 +176,9 @@ php artisan up
 |`APP_NAME`|The gallery name|`Lychee`|
 |`APP_ENV`|Environment of your gallery, `production` or `development`|`production`|
 |`APP_DEBUG`|Enable debug mode|`false`|
-|`APP_URL`|The URL of your gallery|`http://localhost`|
+|`APP_URL`|The URL of your gallery (under which the `public/` folder is accessible)|http://localhost`|
```

Comment:
One of the native speakers (@kamil4 or @d7415) should have a look at this one. The preposition "under" sounds very German to my ears. I  would have written something like

```suggestion
|`APP_URL`|The URL of your gallery (which resolves to the `public/` folder)|http://localhost`|
```

But there are probably better alternatives.

Reply:
I prefer yours to the original. Nothing better is coming to mind.